src/applications/claims-status/components/ClaimsBreadcrumbs.jsx

```diff
@@ -27,7 +27,9 @@ class ClaimsBreadcrumbs extends React.Component {
 
   render() {
     return (
-      <Breadcrumbs>{this.renderBreadcrumbs(this.props.children)}</Breadcrumbs>
+      <VaBreadcrumbs>
+        {this.renderBreadcrumbs(this.props.children)}
```

Comment:
You should be able to use `va-breadcrumbs` here, no need for the `react-bindings` version

Reply:
@jerekshoe fixed it.